assets/svg/app/change-positive.svg

```diff
@@ -1,3 +1 @@
-<svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M4 2L7 6H1L4 2Z" fill="#7FD482"/>
-</svg>
+<svg fill="none" height="4" viewBox="0 0 6 4" width="6" xmlns="http://www.w3.org/2000/svg"><path d="m2.7534.355935c.11931-.172203.37389-.172203.4932-.000001l2.02526 2.923216c.13785.19896-.00455.47085-.2466.47085h-4.050524c-.242046 0-.384443-.27189-.246599-.47085z" fill="#66dd84"/></svg>
```

Comment:
This color is different from the one we have set for `primaryGreen`: `#7FD482`

Reply:
fixed